packages/backend/src/projectAdapters/dbtBaseProjectAdapter.ts

```diff
@@ -143,16 +147,18 @@ export class DbtBaseProjectAdapter implements ProjectAdapter {
                 Logger.debug(
                     'Get warehouse catalog after missing catalog error',
                 );
-                this.warehouseCatalog = await this.warehouseClient.getCatalog(
+                const warehouseCatalog = await this.warehouseClient.getCatalog(
```

Comment:
In our previous cached method, we had a refresh to update teh cache. in here... when is it going to be refreshed ? how often do we need to update the warehouse schema ? @owlas

Reply:
For now it's pretty lazy, it never requests a warehouse update unless there is a `Missing catalog error` - this happens during compilation if it can't find a table/column it expects to be there. 

Then this line triggers. Later I think we want more control over this